src/shortcodes/wordlift_shortcode_geomap.php

```diff
@@ -24,6 +24,11 @@ function wl_shortcode_geomap_get_places( $post_id = null ) {
 		$related_ids = wl_core_get_related_entity_ids( $post_id, array(
 			'status' => 'publish'
 		) );
+		
+		// Also include current entity
+		if( get_post_type( $post_id ) == Wordlift_Entity_Service::TYPE_NAME ) {
```

Comment:
This can be changed in

    if ( Wordlift_Entity_Service->get_instance()->is_entity( $post_id ) ) {